staging/src/k8s.io/client-go/rest/config_test.go

```diff
@@ -277,10 +277,10 @@ func (n *fakeNegotiatedSerializer) DecoderToVersion(serializer runtime.Decoder,
 }
 
 var fakeDialFunc = func(ctx context.Context, network, addr string) (net.Conn, error) {
-	return nil, fakeDialerError
+	return nil, ErrorFakeDialer
 }
 
-var fakeDialerError = errors.New("fakedialer")
+var ErrorFakeDialer = errors.New("fakedialer")
```

Comment:
> error var fakeDialerError should have name of the form errFoo

maybe `errFakeDialer`. A variable starting with a capital letter is public, which is not necessary here